src/app/routes/cpsAsset/utils/getAdditionalPageData.js

```diff
@@ -31,7 +29,7 @@ const getRecommendations = (service, assetUri) => {
       {
         name: 'recommendations',
         attachAgent: true,
-        path: getRecommendationsUrl({ assetUri }),
+        path: getRecommendationsUrl({ assetUri, engine: 'unirecs_datalab' }),
```

Comment:
Are there any services left still using camino? If so we need to add a conditional for those that are using unirecs and those that are still using camino?